app/containers/markdown/new/TaskList.tsx

```diff
@@ -16,7 +17,14 @@ const TaskList = ({ value = [] }: ITasksProps) => {
 		<View>
 			{value.map(item => (
 				<View style={styles.row}>
-					<Text style={[styles.text, { color: colors.bodyText }]}>{item.status ? '- [x] ' : '- [ ] '}</Text>
+					<Text style={[styles.text, { color: colors.bodyText }]}>
+						<CustomIcon
+							testID={item.status ? 'send-to-channel-checked' : 'send-to-channel-unchecked'}
+							name={item.status ? 'checkbox-checked' : 'checkbox-unchecked'}
+							size={24}
+							color='#9297a2'
```

Comment:
Don't put the color directly in the code, use a constant, preferably one that is in the app's const colors, as it doesn't change color, add it as a fixed color.